Trimble.Modus.Components/Controls/TMSegmentedControl/TMSegmentedItem.xaml

```diff
@@ -1,8 +1,8 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Getting this error in the console

`Microsoft.Maui.Controls.BindableObject: Warning: Cannot convert [Color: Red=0, Green=0.3882353, Blue=0.6392157, Alpha=1] to type 'Microsoft.Maui.Controls.Style'
Microsoft.Maui.Controls.BindableObject: Warning: Cannot convert [Color: Red=0, Green=0.3882353, Blue=0.6392157, Alpha=1] to type 'Microsoft.Maui.Controls.Style'
Microsoft.Maui.Controls.BindableObject: Warning: Cannot convert [Color: Red=0, Green=0.3882353, Blue=0.6392157, Alpha=1] to type 'Microsoft.Maui.Controls.Style'
Microsoft.Maui.Controls.BindableObject: Warning: Cannot convert [Color: Red=0, Green=0.3882353, Blue=0.6392157, Alpha=1] to type 'Microsoft.Maui.Controls.Style'

`

Reply:
To check later. @BAL4-R Can you create a tech debt issue for this? We can close this one for now.